File: SE226_Lab1/main.py
name = str(input("What is your name?"))
print("Hello " + name)
ID = str(input("What is your student ID"))
print("Your ID is " + ID)

number1 = int(input("Enter a number."))
number2 = int(input("Enter another number."))
sum = int(number1 + number2)
diff = int(number1 - number2)
prod = int(number1 * number2)
print("Summation of the numbers is " + str(sum))
print("Difference of the numbers is " + str(diff))
print("Product of the numbers is " + str(prod))

for i in range(0, 3):
    for j in range(0, i+1):
        print("*", end='')
    print("\r")

File: SE226_Lab1/main.cpp

#include <iostream>
using namespace std;


int main()
{
    string name;
    string ID;

    cout << "What is your name?" << endl;
    cin >> name;
    cout << "Hello " << name << " !" << endl;
    cout << "What's your student ID?" << endl;
    cin >> ID;
    cout << "Your ID is " << ID << endl;

int number1,number2,sum,diff,prod;

cout << "Enter a number." << endl;
cin >> number1;
cout << "Enter another number." << endl;
cin >> number2;
sum = number1 + number2;
diff = number1 - number2;
prod = number1 * number2;
cout << "Summation of the numbers is " << sum << endl << "Difference of the numbers is " << diff << endl<<
"Product of the numbers is " << prod << endl;


    for(int i = 1; i <= 3; i++){
        for(int j = 1; j <= i; j++)
        {
            cout << "* ";
        }
        cout << "\n";
    }
    return 0;
}